### PR TITLE
DDF-2824 updates to feature installation docs

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates documentation for application reference sections.

#### Who is reviewing it? 
@mcalcote @ahoffer @garrettfreibott 

#### Choose 2 committers to review/merge the PR. 
@lessarderic
@ricklarsen - Documentation
@rzwiefel
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

content changes only

#### What are the relevant tickets?
[DDF-2824](https://codice.atlassian.net/browse/DDF-2824)

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_appReferences/mg-solr.adoc [hunk 1]
```diff
@@ -17,4 +17,9 @@ To use the ${ddf-solr} Application, the following apps/features must be installe
 
 === Installing ${ddf-solr}
 
-The ${ddf-solr} Application is installed by default with a standard installation.
+Install ${ddf-solr} through the ${admin-console}.
+
+. Navigate to the *${admin-console}*.
+. Select the *System* tab.
+. Select the *Features* tab.
+. Install the `registry-app` feature.
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply distribution/docs/src/main/resources/content/_appReferences/mg-solr.adoc]: oops. fixed

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by vinamartin** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
rebased to rerun build now

**Pull request merged**